coderd/workspaceagentsrpc.go

```diff
@@ -96,6 +103,10 @@ func (api *API) workspaceAgentRPC(rw http.ResponseWriter, r *http.Request) {
 	ctx, wsNetConn := websocketNetConn(ctx, conn, websocket.MessageBinary)
 	defer wsNetConn.Close()
 
+	ycfg := yamux.DefaultConfig()
+	ycfg.LogOutput = nil
+	ycfg.Logger = slog.Stdlib(ctx, logger.Named("yamux"), slog.LevelDebug)
```

Comment:
yamux only logs ERR and WARN, so I think it's appropriate to bump this to `LevelInfo` so they will show up in coderd logs by default.